contribs/gmf/apps/appmodule.js

```diff
@@ -15,4 +15,4 @@ goog.require('gmf');
 /**
  * @type {!angular.Module}
  */
-var appModule = angular.module('app', [gmfModule.name]);
+var appModule = angular.module('app', [gmfModule.name, 'ngTouch']);
```

Comment:
Why not but:
* This file is shared with the desktop app
* Done like this (As I understand) it don't add any code in the build file
* This will be copy in the client application => more difficult to maintain...